redis/cluster.py

```diff
@@ -2190,8 +2195,8 @@ def _send_cluster_commands(
         # buffered result back from a previous command and
         # every single request after to that connection will always get
         # a mismatched result.
-        for n in nodes.values():
-            n.connection_pool.release(n.connection)
+        # for n in nodes.values():
```

Comment:
Can you delete this lines from the comment and move the explanations of this lines before them?

Reply:
I already fixed this. could you review it again?